conclave-enclave/src/main/kotlin/com/r3/conclave/enclave/internal/GramineEntryPoint.kt

```diff
@@ -3,33 +3,32 @@ package com.r3.conclave.enclave.internal
 import com.r3.conclave.common.EnclaveMode
 import com.r3.conclave.common.internal.CallHandler
 import com.r3.conclave.common.internal.EnclaveCallType
+import com.r3.conclave.common.internal.PluginUtils.PYTHON_FILE
 import com.r3.conclave.enclave.Enclave
 import com.r3.conclave.utilities.internal.getRemainingString
 import java.lang.reflect.InvocationTargetException
 import java.lang.reflect.Method
 import java.nio.ByteBuffer
 import java.nio.file.Path
+import kotlin.system.exitProcess
+import java.lang.Boolean.parseBoolean
 import java.nio.file.Paths
 import java.util.*
-import kotlin.system.exitProcess
 
 object GramineEntryPoint {
     private const val USAGE_STRING = "usage: GramineEntryPoint <port>"
-    private const val ENCLAVE_METADATA_FILE = "enclave-metadata.properties"
     private const val EXIT_ERR = -1
 
-    /** Enclave metadata variables */
-    private var isSimulation: Boolean? = null
-    private var signingKeyMeasurement: ByteArray? = null
-    private var pythonEnclaveScript: Path? = null
+    /** Enclave metadata, retrieved from the manifest. */
+    private val isSimulation = parseBoolean(System.getenv("CONCLAVE_IS_SIMULATION_ENCLAVE")!!)
+    private val isPythonEnclave = parseBoolean(System.getenv("CONCLAVE_IS_PYTHON_ENCLAVE")!!)
```

Comment:
`Boolean.getBoolean` - very badly named but it does what you're doing here.

Reply:
Don't these get system properties? We need to get environment variables.